cds_ils/cli.py

```diff
@@ -198,3 +241,290 @@ def run_command(command, catch_exceptions=False, verbose=True):
     run_command("vocabulary index json --force {}".format(json_files))
     run_command("vocabulary index opendefinition spdx --force")
     run_command("vocabulary index opendefinition opendefinition --force")
+
+
+@click.group()
+def demo():
+    """Create real demo data."""
+
+
+def mint_record_pid(pid_type, pid_field, record):
+    """Mint the given PID for the given record."""
+    PersistentIdentifier.create(
+        pid_type=pid_type,
+        pid_value=record[pid_field],
+        object_type="rec",
+        object_uuid=record.id,
+        status=PIDStatus.REGISTERED,
+    )
+    db.session.commit()
+
+
+@demo.command()
+@click.option("--json-path", "docs_path")
+@with_appcontext
+def create_demo_docs(docs_path):
+    """Insert real demo docs."""
+    indexer = RecordIndexer()
+
+    demo_docs = os.path.join(CURRENT_DIR, docs_path)
+    with open(os.path.join(demo_docs)) as f:
+        demo_data = json.loads(f.read())
+
+    for d in demo_data:
+        doc = Document.create(d)
+        mint_record_pid(DOCUMENT_PID_TYPE, "pid", doc)
+        doc.commit()
+        db.session.commit()
+        indexer.index(doc)
+
+    click.secho("Docs were created successfully.", fg="blue")
+
+
+def get_pids(search):
+    """Get a list of pids of the search results."""
+    s = search()
+    results = s.execute()
+    pids = []
+    for hit in results.hits:
+        pids.append(hit["pid"])
+
+    return pids
+
+
+@demo.command()
+@click.option("--json-path", "items_path")
+@with_appcontext
+def create_demo_items(items_path):
+    """Insert real demo items."""
+    indexer = RecordIndexer()
+
+    demo_items = os.path.join(CURRENT_DIR, items_path)
+    with open(os.path.join(demo_items)) as f:
+        demo_items = json.loads(f.read())
+
+    intloc_pids = get_pids(InternalLocationSearch)
+
+    for i in demo_items:
+        i["internal_location_pid"] = random.choice(intloc_pids)
+        item = Item.create(i)
+        mint_record_pid(ITEM_PID_TYPE, "pid", item)
+        item.commit()
+        db.session.commit()
+        indexer.index(item)
+
+    click.secho("Items were created successfully.", fg="blue")
+
+
+@demo.command()
+@click.option("--user-email", "user_email")
+@click.option("--is-past-loan", is_flag=True)
+@with_appcontext
+def create_loan(user_email, is_past_loan):
+    """Create a loan."""
+    patron = User.query.filter_by(email=user_email).one_or_none()
```

Comment:
```suggestion
    patron = User.query.filter_by(email=user_email).one()
```
I think we should fail if not found...